### PR TITLE
profile update: added some info

### PR DESCRIPTION
**Pull request by kalyan-vurugonda**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2936"></a>

### CODE REVIEW

**Comment by kalyan-vurugonda** [comment]
@Pradumnasaraf @eddiejaoude 
Can you please look into this conflict.

**Comment by ChinmayMhatre** [comment]
Hey @kalyan-vurugonda , I see the CHANGELOG.md is still added to the PR.

**Comment by kalyan-vurugonda** [comment]
> Hey @kalyan-vurugonda , I see the CHANGELOG.md is still added to the PR.

Ok, I will check it.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,3 +2,4 @@
 
 
 
+
```
Comment (Pradumnasaraf) [inline CHANGELOG.md:5]:
```suggestion
```

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**